src/components/accordion/accordion.test.tsx

```diff
@@ -0,0 +1,456 @@
+import React from "react";
+import { act } from "react-dom/test-utils";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { StyledAccordionHeadingsContainer } from "./accordion.style";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+import useResizeObserver from "../../hooks/__internal__/useResizeObserver";
+import Textbox from "../textbox";
+import { Accordion } from ".";
+import AccordionGroup from "./accordion-group/accordion-group.component";
+import Logger from "../../__internal__/utils/logger";
+
+jest.mock("../../hooks/__internal__/useResizeObserver");
+
+// mock Logger.deprecate and console.error so that no console warnings occur while running the tests
+const loggerSpy = jest.spyOn(Logger, "deprecate").mockImplementation(() => {});
+const consoleSpy = jest.spyOn(console, "error").mockImplementation(() => {});
+
+afterEach(() => {
+  loggerSpy.mockReset();
+});
+
+afterAll(() => {
+  loggerSpy.mockReset();
+  consoleSpy.mockReset();
+});
+
+describe("Accordion", () => {
+  it("should display deprecation warning for `scheme` prop once", () => {
+    render(<Accordion scheme="transparent" title="Title" />);
+
+    expect(loggerSpy).toHaveBeenCalledWith(
+      "The `scheme` prop for `Accordion` component is deprecated and will soon be removed."
+    );
+    expect(loggerSpy).toHaveBeenCalledTimes(1);
```

Comment:
**question(non-blocking):** If we have only one Accordion component won't the loggerSpy only be called once anyway?

Reply:
🤷 I put this in because the description includes the word "once". I'll want to remove that word if I remove that assertion - I don't think it makes much of a difference so happy to do it either way.

Reply:
It's fine. I have had a look elsewhere and we've done this same exact assertion with the same kind of wording. Keep it as is which means it'll be consistent.